contract_manager/scripts/check_proposal.ts

```diff
@@ -134,6 +136,59 @@ async function main() {
           }
         }
       }
+      if (instruction.governanceAction instanceof EvmExecute) {
+        // Note: it only checks for upgrade entropy contracts right now
+        console.log(
+          `Verifying EVMExecute Contract on ${instruction.governanceAction.targetChainId}`
+        );
+        for (const chain of Object.values(DefaultStore.chains)) {
+          if (
+            chain instanceof EvmChain &&
+            chain.wormholeChainName ===
+              instruction.governanceAction.targetChainId
+          ) {
+            const executorAddress =
+              instruction.governanceAction.executorAddress;
+            const callAddress = instruction.governanceAction.callAddress;
+            const calldata = instruction.governanceAction.calldata;
+
+            // currently executor is only being used by the entropy contract
+            const contract = new EvmEntropyContract(chain, callAddress);
+            const owner = await contract.getOwner();
+            console.log(
+              `Executor Address: ${executorAddress.toUpperCase()} is equal to Owner Address: ${owner
```

Comment:
I would only log an error if there is a mismatch. We show the digests since council need to verify them manually but this stage can be done automatically.